injected/src/config-feature.js

```diff
@@ -19,6 +26,7 @@ export default class ConfigFeature {
     /**
      * @type {{
      *   debug?: boolean,
+     *   platform: import('./utils.js').Platform,
```

Comment:
We'd have to check if Autofill passes this btw. Maybe it's ok not to support it there.

Reply:
As we've got so few supporting platforms I think we're good here. Let's just add them adhoc.